test/test_helper.rb

```diff
@@ -17,7 +17,7 @@
 require 'mocha/minitest'
 require 'hammer_cli'
 
-KATELLO_VERSION = Gem::Version.new(ENV['TEST_API_VERSION'] || '3.17')
+KATELLO_VERSION = Gem::Version.new(ENV['TEST_API_VERSION'] || '3.18')
```

Comment:
Since this is going to be cherry-picked downstream I'm not sure if this will break anything since it's part of this PR? Maybe it's fine and only related to the tests. Ditto my comment for the above JSON

Reply:
I will make a separate pr to update the test data that way we are not updating it with this one

Reply:
Made https://github.com/Katello/hammer-cli-katello/pull/744 to separate out the test helper and new api data